text/0119-instance-options.md

```diff
@@ -0,0 +1,454 @@
+./meilisearch --db-path
+
+error: The argument '--db-path <DB_PATH>' requires a value but none was supplied
+```
+
+✅ Correct
+
+```bash
+./meilisearch --db-path ./meilifiles
+```
+
+2. Some command-line options take an implicit boolean as a value. In this case, the users should not set any value when using the option.
+
+❌ Wrong
+
+```bash
+./meilisearch --schedule-snapshot yes
+
+error: Found argument 'yes' which wasn't expected, or isn't valid in this context
+```
+
+✅ Correct
+
+```bash
+./meilisearch --schedule-snapshot
+```
+
+The expected behavior of each flag is described in the list above.
+
+### Exhaustive list of options
+
+- [Database path](#database-path)
+- [Environment](#environment)
+- [HTTP address & port binding](#http-address-port-binding)
+- [Master key](#master-key)
+- [Disable analytics](#disable-analytics)
+- [Dumps](#dumps-destination)
+- [Dumps destination](#dumps-destination)
+- [Import dump](#import-dump)
+- [Log level](#log-level)
+- [Max index size](#max-index-size)
+- [Max TASK_DB size](#max-task-db-size)
+- [Payload limit size](#payload-limit-size)
+- [Snapshots](#schedule-snapshot-creation):
+- [Schedule snapshot creation](#schedule-snapshot-creation)
+- [Snapshot destination](#snapshot-destination)
+- [Snapshot interval](#snapshot-interval)
+- [Import snapshot](#import-snapshot)
+- [Ignore missing snapshot](#ignore-missing-snapshot)
+- [Ignore snapshot if DB exists](#ignore-snapshot-if-db-exists)
+- [SSL configuration](#ssl-authentication-path):
+- [SSL authentication path](#ssl-authentication-path)
+- [SSL certificates path](#ssl-certificates-path)
+- [SSL key path](#ssl-key-path)
+- [SSL OCSP path](#ssl-ocsp-path)
+- [SSL require auth](#ssl-require-auth)
+- [SSL resumption](#ssl-resumption)
+- [SSL tickets](#ssl-tickets)
+- [Database path](#database-path)
+- [Environment](#environment)
+- [HTTP address & port binding](#http-address-port-binding)
+- [Master key](#master-key)
+- [Disable analytics](#disable-analytics)
+- [Dumps](#dumps-destination)
+- [Dumps destination](#dumps-destination)
+- [Import dump](#import-dump)
+- [Log level](#log-level)
+- [Max index size](#max-index-size)
+- [Max TASK_DB size](#max-task-db-size)
+- [Payload limit size](#payload-limit-size)
+- [Snapshots](#schedule-snapshot-creation)
+- [Schedule snapshot creation](#schedule-snapshot-creation)
+- [Snapshot destination](#snapshot-destination)
+- [Snapshot interval](#snapshot-interval)
+- [Import snapshot](#import-snapshot)
+- [Ignore missing snapshot](#ignore-missing-snapshot)
+- [Ignore snapshot if DB exists](#ignore-snapshot-if-db-exists)
+- [Memory usage](#memory-usage)
+- [Indexing jobs](#indexing-jobs)
+- [SSL configuration](#ssl-authentication-path)
+- [SSL authentication path](#ssl-authentication-path)
+- [SSL certificates path](#ssl-certificates-path)
+- [SSL key path](#ssl-key-path)
+- [SSL OCSP path](#ssl-ocsp-path)
+- [SSL require auth](#ssl-require-auth)
+- [SSL resumption](#ssl-resumption)
+- [SSL tickets](#ssl-tickets)
```

Comment:
```suggestion
- [Database path](#database-path)
- [Environment](#environment)
- [HTTP address & port binding](#http-address-port-binding)
- [Master key](#master-key)
- [Disable analytics](#disable-analytics)
- [Dumps](#dumps-destination)
- [Dumps destination](#dumps-destination)
- [Import dump](#import-dump)
- [Log level](#log-level)
- [Max index size](#max-index-size)
- [Max TASK_DB size](#max-task-db-size)
- [Payload limit size](#payload-limit-size)
- [Snapshots](#schedule-snapshot-creation)
- [Schedule snapshot creation](#schedule-snapshot-creation)
- [Snapshot destination](#snapshot-destination)
- [Snapshot interval](#snapshot-interval)
- [Import snapshot](#import-snapshot)
- [Ignore missing snapshot](#ignore-missing-snapshot)
- [Ignore snapshot if DB exists](#ignore-snapshot-if-db-exists)
- [Memory usage](#memory-usage)
- [Indexing jobs](#indexing-jobs)
- [SSL configuration](#ssl-authentication-path)
- [SSL authentication path](#ssl-authentication-path)
- [SSL certificates path](#ssl-certificates-path)
- [SSL key path](#ssl-key-path)
- [SSL OCSP path](#ssl-ocsp-path)
- [SSL require auth](#ssl-require-auth)
- [SSL resumption](#ssl-resumption)
- [SSL tickets](#ssl-tickets)
```

Reply:
Removing duplicated value here. I've kept the list with the newly added options.